scripts/maintenance/audit_branches.py

```
@@ -0,0 +1,101 @@
+    try:
+        result = subprocess.run(
+            args,
+            shell=False,
+            check=False,  # We handle errors based on returncode or empty output
+            capture_output=True,
+            text=True
+        )
+        if result.returncode == 0:
+            return result.stdout.strip()
+        return None
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)